hud-data/budhud.json

```diff
@@ -8,9 +8,24 @@
     "discord": "TkxNKU2",
     "album": "https://imgur.com/a/aJ1K5"
   },
+  "flags": ["centered", "customizations", "streamer mode", "custom crosshairs"],
```

Comment:
flags cannot contain spaces, should be `streamer-mode`

`customizations` should be `customization`

`custom crosshairs` should be `crosshairs`

`centered` is interesting! but maybe it should be a `positioning` trait?

Reply:
Note: flags should make sense to say "has x"